unit_tests/test_redis_aggregator.py

```diff
@@ -145,9 +145,9 @@ def test_constructor_connect_and_subscribe_side_effects_3():
         -> Side effects of _subscribe_to_response_channels private method
         """
         aggregator = RedisAggregator(aggregator_name=TEST_AGGREGATOR_NAME)
-        channel_dict_2 = {f"external-aggregator/*/{aggregator.aggregator_uuid}/events"
+        channel_dict_2 = {f"external-aggregator//{aggregator.aggregator_uuid}/events"
                           f"/all": aggregator._events_callback_dict,
-                          f"external-aggregator/*/{aggregator.aggregator_uuid}/response"
```

Comment:
Thank you for the verbose answer! I still wonder though: What does the absence of the `*` mean? 
Doesn't it also mean:
> the aggregator subsribes to all responses from all simulations

?

Reply:
Sorry took a long time to check it out. Actually the * is not absent, it is just replaced with the actual simulation id, therefore each SDK instance will connect only to one simulation with this specified ID. What you describe here :

> the aggregator subsribes to all responses from all simulations

was the former behavior, because the * in the channel name subscribed to all responses from all simulations. Now that * is replaced with the actual simulation id , the aggregator subscribes to all responses from one specific simulation.